docker/images/docker-spark/master/supervisord.conf

```diff
@@ -18,7 +19,7 @@ autostart=true
 autorestart=true
 process_name=metadata_ingestion
 numprocs=1
-stdout_logfile=/dev/fd/1
+stdout_logfile=/dev/stdout
```

Comment:
Changed to stdout because it's more clear as to what is going on but does the same thing